Sources/DistributedActors/ActorID.swift

```diff
@@ -197,17 +231,27 @@ extension ClusterSystem {
             self.context = context
             self._location = .remote(node)
             self.incarnation = incarnation
-            self.tags = ActorTags()
-            // TODO: avoid mangling names on every spawn?
-            if let mangledName = _mangledTypeName(type) {
-                self.tags[ActorTags.type] = .init(mangledName: mangledName)
+            if let mangledName = _mangledTypeName(type) { // TODO: avoid mangling names on every spawn?
+                self.context.metadata[ActorMetadata.type] = .init(mangledName: mangledName)
             }
             traceLog_DeathWatch("Made ID: \(self)")
         }
 
         internal var withoutContext: Self {
```

Comment:
Should this be renamed to `withoutLifecycle`? Or make `metadata` `nil` as well?

Reply:
Good one, thanks -- it kind of is "without stuff ... but keep the tags" hm, ill rename for now